Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -680,7 +681,7 @@ Don't have a 4chan Pass?<br>
     <string name="settings_experimental_settings_title">⚠ EXPERIMENTAL ⚠</string>
     <string name="settings_experimental_settings_description">These settings may be in an unstable state. They are for testing. They may break the app and you may have to reset the app data. Use them at your own risk if you are really interested in some feature. Backup everything beforehand.</string>
     <string name="incremental_thread_downloading_title">Incremental thread downloading</string>
-    <string name="incremental_thread_downloading_description">Allows you to download threads to view them after the original thread dies</string>
+    <string name="incremental_thread_downloading_description">Allows you to download threads to view them after the original thread dies or when you have no internet connection. Basically, this thread will be stored locally along with all the files.</string>
```

Comment:
Remove the "basically" part. Not needed.

Reply:
Done.